src/client/opamCommands.ml

```diff
@@ -1535,7 +1535,7 @@ let update =
        changes, 1 if there were none. Repositories or development packages \
        that failed to update are considered without changes. With \
        $(b,--upgrade), behaves like $(b,opam upgrade --check), that is, \
-       returns 0 only if there are currently availbale updates." in
+       returns 0 only if there currently are available updates." in
```

Comment:
Ouch, this description was not wonderful before. The key point is that with `--upgrade`, `--check`'s status refers to available *upgrades* rather than reporting whether there were upstream *updates*, so the change should also rename the word *updates* to *upgrades*. How about replacing the whole sentence (from `With \` onwards) with:

```ocaml
With \
$(b,--upgrade), applies to the upgrade step, that is $(b,opam update --upgrade --check) \
behaves like $(b,opam update ; opam upgrade --check), returning 0 if there are available \
upgrades, rather than upstream updates."
```